test/utils/axios-helper.test.js

```diff
@@ -6,16 +6,21 @@ const config = require('../../lib/config');
 const auth = require('../../lib/auth');
 const { setPiiToken } = require('../../lib/utils/axios-helpers');
 const { getTestAccessToken } = require('../test-helpers/auth-helpers');
+const { mockConfig } = require('../test-helpers/mock-config');
 
 const RoUrl = faker.internet.url();
 const piiUrl = faker.internet.url();
```

Comment:
Nit (optional): May as well use the same approach in L11–12 as in L13–14
```suggestion
const apiServerUrl = faker.internet.url();
const piiServerUrl = faker.internet.url();
```